apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/logs/_components/DocumentLogs/DocumentLogInfo/index.tsx

```diff
@@ -27,24 +28,42 @@ export function DocumentLogInfo({
   providerLogs,
   isLoading = false,
   error,
+  className,
 }: {
   documentLog: DocumentLogWithMetadata
   providerLogs?: ProviderLogDto[]
   isLoading?: boolean
   error?: Error
+  className?: string
 }) {
   const [selectedTab, setSelectedTab] = useState<string>('metadata')
+  const ref = useRef<HTMLDivElement>(null)
+  const height = useDynamicHeight({ ref, paddingBottom: 16 })
```

Comment:
Oh man, web apps with scrollable areas are not as easy as doing web pages. I don't care what DHH says